### ISSUE TITLE
It doesn't lower or turn up the zoom audio

### ISSUE THREAD

**Issue by Jumimi** [kyleneideck/BackgroundMusic#527]
Hi,
Basically it just doesn't do anything to zoom audio, it doesn't lower or turn up the audio. (Edit: typo)

**Comment by Vermats** [comment]
Yeah same happened to me with Microsoft teams but it seems like it sees the app window and the meet window as different entities and the meet not actually being the same window as the app,has its media control scroll in the "more apps" section below the running apps. I hope this helps

**Comment by ForsakeLJK** [comment]
same here 🙋😖

**Comment by tylerharpool** [comment]
+1

**Comment by MattUnderscoreZhang** [comment]
Same issue for me. I couldn't find a separate mixer for Zoom under "more apps".

**Comment by heismauri** [comment]
You should select the Background Music output, at least that worked for me
![Screen Shot 2021-09-14 at 5 41 22 PM](https://user-images.githubusercontent.com/47569205/133330808-790ce402-a66e-41bc-aca7-34c84e5ecdf9.png)

**Comment by Khazbs** [comment]
Background Music is selected as Speaker in Microsoft Teams, but the app volume slider still doesn't affect the meeting volume. Running version `0.4.0-SNAPSHOT-b38f6dd` on macOS Monterey `12.2.1 (21D62)`.
